AppCUI/src/Terminal/SDLTerminal/SDLTerminalScreen.cpp

```diff
@@ -88,19 +88,21 @@ bool SDLTerminal::initFont(const InitializationData& initData)
     default:
         break;
     }
-    TTF_Init();
+
+    CHECK(TTF_WasInit() || TTF_Init() == 0,
+          false,
+          (std::string("Failed to initialize true type support: ") + TTF_GetError() + "\n").c_str());
```

Comment:
De ce avem neboie de acel std::string ?
Nu mergea cu "Failed to initialize true type support: %s\n",TTF_GetError()

Reply:
ba da, my bad